database/GenerateListingsPG.js

```diff
@@ -0,0 +1,59 @@
+const fs = require('fs');
+const faker = require('faker');
+// ================= FS Writable Stream Attempt =========
+const wStream = fs.createWriteStream('./postgresListings.csv');
+const getRandomInt = (min, max) => {
+  const minimum = Math.ceil(min);
+  const maximum = Math.floor(max);
+  return Math.floor(Math.random() * (maximum - minimum)) + minimum;
+};
+
+let listingCount = 1;
+const start = new Date().getTime();
+const sentences = [];
+const boolean = [true, false];
+const locations = [];
+const names = [];
+const n = 10000000;
+
+for (let j = 0; j < 100; j += 1) {
```

Comment:
I would recommend wrapping the bodies of scripts like this in functions for modularity and all that goodness. But I'd be curious about John's opinion too.